cluster-up/cluster/kind/common.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e
 
+function detect_cri() {
+    if podman ps >/dev/null 2>&1; then echo podman; elif docker ps >/dev/null 2>&1; then echo docker; fi
```

Comment:
Since we can't use kind with podman via a non root user, even if the socket belong to the root user,
It just adds complexity.
Moreover, it enforce the root user to start a socket, where with the current approach podman installed is enough.
The socket is not needed in all cases.
Moreoever the default socket path is not at `XDG_RUNTIME_DIR`, but at `/run/podman/podman.socket` btw

Only when we do support non root user via a socket (if we do, for example there might be SR-IOV commands that need root maybe), It will be good to use the socket path as well.
Otherwise It will just add restrictions and complexity atm.

Reply:
OK  thanks for the good explanation - I am happy with the existing check.